personio-personnel-data-api-oa3.yaml

```diff
@@ -3983,3 +4105,700 @@ components:
+          example: "2019-12-01"
+        end_date:
+          type: string
+          format: date
+          example: "2019-12-01"
+        half_day_at_start:
+          type: boolean
+        half_day_at_end:
+          type: boolean
+        employee_id:
+          type: integer
+          example: 17
+
+    DurationAttribute:
+      type: object
+      properties:
+        attribute_id:
+          type: string
+          example: duration_123
+        data_type:
+          type: string
+          enum:
+            - DURATION
+        duration:
+          type: integer
+          example: 10
+        employee_id:
+          type: integer
+          example: 17
+
+    PerformanceTargetAttribute:
+      type: object
+      properties:
+        attribute_id:
+          type: string
+          example: performance_target_123
+        data_type:
+          type: string
+          enum:
+            - PERFORMANCE_TARGET
+        employee_id:
+          type: integer
+          example: 17
+        performance_targets:
+          type: array
+          items:
+            $ref: "#/components/schemas/PerformanceTarget"
+
+    PerformanceTarget:
+      type: object
+      properties:
+        performance_target_id:
+          type: string
+          example: 1
+        performance_target_value:
+          type: string
+          example: 20
+
+    PerformanceKpiAttribute:
+      type: object
+      properties:
+        attribute_id:
+          type: string
+        data_type:
+          type: string
+          enum:
+            - PERFORMANCE_KPI
+        employee_id:
+          type: integer
+          example: 17
+        performance_target_kpis:
+          type: array
+          items:
+            $ref: "#/components/schemas/PerformanceTarget"
+
+    PerformanceTargetKpi:
+      type: object
+      properties:
+        performance_target_id:
+          type: string
+          example: id_123
+        performance_kpis:
+          type: array
+          items:
+            $ref: "#/components/schemas/PerformanceKpi"
+
+    PerformanceKpi:
+      type: object
+      properties:
+        performance_kpi_id:
+          type: string
+          example: kpi_123
+        performance_kpi_value:
+          type: string
+          example: 14
+
+    SalaryAttribute:
+      type: object
+      properties:
+        attribute_id:
+          type: string
+          example: salary_123
+        data_type:
+          type: string
+          enum:
+            - SALARY
+        amount:
+          type: string
+          example: 12.20
+        currency_symbol:
+          type: string
+          example: $
+        employee_id:
+          type: integer
+          example: 17
+
+    CompensationAttribute:
+      type: object
+      properties:
+        attribute_id:
+          type: string
+          example: ABCXXX
+        data_type:
+          type: string
+          enum:
+            - COMPENSATION
+        amount:
+          type: string
```

Comment:
Shouldn't the amount be double/float?

Reply:
@raul-mdelfin yes, these values are stored as strings.